### PR TITLE
List Channels

### PR DESCRIPTION
**Pull request by valevassallo**
- Add Channels component
- Add ListChannels component which creates a list with the channels' names
- Install @emotion/styled and @emotion/core (because of peer dependence)

### CODE REVIEW

File: app/src/components/ListChannels.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import styled from "@emotion/styled";
+
+function ListChannels(props) {
+  const elementsChannel = props.channels.map(channel => <li># {channel}</li>);
+
+  const List = styled.ul`
```
Comment (yummta) [inline app/src/components/ListChannels.js]: We could add globals styles to reset some tags. Are you agree?

Comment (valevassallo) [reply app/src/components/ListChannels.js]: What does "reset some tags" mean?

Comment (yummta) [reply app/src/components/ListChannels.js]: Some tags like ul, p, h(1,2,3...) have default styles. For example, every time we use <ul> we need to put "list-style: none;" to clean dots. We could put this styles only once for all <ul>.

Comment (valevassallo) [reply app/src/components/ListChannels.js]: Oh, okay. I agree. Next time I add something, I'll make sure to add it.

Comment (valevassallo) [reply app/src/components/ListChannels.js]: Oh, okay. I agree. Next time I upload something, I'll make sure to add it.

**Pull request merged**